tests/test_vault.yml

```diff
@@ -61,6 +64,18 @@
               ip_sans: "127.0.0.1"
               alt_names: "example.com"
               exclude_cn_from_sans: true
+    - name: Configure PKI
```

Comment:
You're hitting an ansible quirk where multiple import_role tasks for the same role in one play will have the task variables from the last import overriding earlier imports. The solution is to use include_role instead, which does a dynamic include and works as expected.

Reply:
The name of the task should be changed to reflect the difference between the two invocations of the role.

Reply:
Done.